test/dynamics/common.py

```diff
@@ -101,6 +106,32 @@ def jit_grad_wrap(self, func_to_test: Callable) -> Callable:
         return wf(f)
 
 
+class TestDiffraxBase(unittest.TestCase):
+    """Base class with setUpClass and tearDownClass for importing diffrax solvers
+
+    Test cases that inherit from this class will automatically work with diffrax solvers
+    backend.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        try:
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
+                # pylint: disable=import-outside-toplevel
```

Comment:
should the `import-outside-toplevel` also be moved to the next line, like:
```
import diffrax  # pylint: disable=unused-import,import-outside-toplevel
```
Not sure if this will disable it for the rest of the file.